### ISSUE TITLE
QuicTransport connect error

### ISSUE THREAD

**Issue by Geshode** [GoogleChrome/samples#692]
I get the following error message, when clicking 'Connect' on the [QuicTransport client sample site](https://googlechrome.github.io/samples/quictransport/client.html).

> Failed to create connection object. ReferenceError: QuicTransport is not defined

How can I fix that? Do I need to register for my own OT token?

@jeffposnick 
@vasilvv

**Comment by jeffposnick** [comment]
The feature is available in Chrome 84+, which should be the [current stable release](https://chromestatus.com/features/schedule), but that might not have rolled out everywhere yet.

If you're still on Chrome 83 or earlier, you could explicit try using [Chrome Canary](https://www.google.com/chrome/canary/).

(I just [updated the OT token last week](https://github.com/GoogleChrome/samples/commit/62eac5960187d22010b5302a5a746e4c6c1cb20d), and confirmed that it's working on Chrome 84+.)

**Comment by Geshode** [comment]
@jeffposnick 

That was the problem. After updating to version 84, the error didn't occur anymore.

When I start the quic_transport_server.py, after following the instructions with creating the certificate and key, and adding the flags to Chrome, I get the following error now:

> Failed to establish a connection to quic-transport://localhost:4433/counter: net::ERR_QUIC_PROTOCOL_ERROR.QUIC_HANDSHAKE_FAILED (TLS handshake failure (ENCRYPTION_HANDSHAKE) 46: certificate unknown).

I searched for the error message, but the only information, which I got about the certificate unknown error, was the following:

> Some other (unspecified) issue arose in processing the certificate, rendering it unacceptable.

From [here](https://tools.ietf.org/html/rfc8446)

What could be the cause for this error?

**Comment by wilaw** [comment]
Same error with Chrome 85.0.4183.121 on the  [QuicTransport client sample site](https://googlechrome.github.io/samples/quictransport/client.html). I tried the default OT token as well as registering my own. Also tried running page under localhost to no avail. Failing at

var transport = new QuicTransport(url);

I have launched chrome with flags to force quic-on on my localhost:443 connection as well as ignoring cert errors for my self-signed cert which the python origin is presenting. 

Is there any reason this sample should no longer be working before I debug on in to the night?
